recipes/libmatio/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+./autogen.sh
```

Comment:
Does it not already come with `configure`? Also, could we add a blank line after `#!/bin/bash` for better readability.